### PR TITLE
Bumped app versionCode

### PR DESCRIPTION
**Pull request by beaufortfrancois**
Once this is reviewed and accepted, could we push this update to the Play Store?

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -8,7 +8,7 @@ android {
         applicationId "io.github.webbluetoothcg.bletestperipheral"
         minSdkVersion 21
         targetSdkVersion 22
-        versionCode 2
+        versionCode 3
         versionName "2.0"
```
Comment (g-ortuno) [inline app/build.gradle]: We probably want to change the version name to.

**Comment by beaufortfrancois** [comment]
I've bumped `versionName` as well based on your feedback.

**Comment by g-ortuno** [comment]
Pushed to the Play Store!

**Comment by beaufortfrancois** [comment]
Perfect! Thank you Giovanni.

**Pull request merged**